cmd/gindoid/main.go

```diff
@@ -66,6 +66,17 @@ Previously generated pages are overwritten, so this command only makes sense if
 		Version:               verstr,
 		DisableFlagsInUseLine: true,
 	}
+	cmds[4] = &cobra.Command{
+		Use:   "make-doi-xml <yml file>...",
```

Comment:
What about instead calling it `make-xml`.  It follows the convention of the other command `make-html` and avoids the repetition of the word `doi` when running `gindoi make-doi-xml`.

Reply:
Sounds good, lets do it.